src/views/Shapes/Capsule.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Modifiers } from "../../utils/modifiers";
+import { Rectangle } from "./Rectangle";
+
+type CapsuleProps = Omit<Modifiers,"backgroundColor"> & {
+    fill: string
```

Comment:
lets make fill optional, and we can use the useUIColor hook to give them default colors
`const UIColor = useUIColor();`

and access the colors like:
`UIColor.systemBlue`

Reply:
this hook takes colorscheme into consideration by default too!

Reply:
Fixed in [8c172e0](https://github.com/andrew-levy/swiftui-react-native/pull/10/commits/8c172e0af1b8b252edf1857088c13ab4c7a7fe59)